user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Configuring_SAML/SAML_using_Entra_ID.md

```diff
@@ -145,7 +145,7 @@ To set up external authentication, you first need to create an enterprise applic
        </md:SPSSODescriptor>
      ```
 
-1. Replace [ENTITYID] with the IP address or the DNS name of your DataMiner System. This must be the same as the *Entity ID* you specified while setting up the Microsoft Entra ID Enterprise application.
+1. Replace [ENTITYID] with the IP address or the DNS name of your DataMiner System, for example ``https://dataminer.example.com/``. This must be the same as the *Entity ID* you specified while setting up the Microsoft Entra ID Enterprise application.
```

Comment:
The example url is already on the line below so I don't know if this would make it any clearer

Reply:
In fact, I think mentioning the same example URL as below here introduces confusion instead of resolving it: first we say to replace something with "https://dataminer.example.com/", then we ask to replace "https://dataminer.example.com/" with something else. Perhaps we could use a different example DNS name as an example here to prevent introducing this confusion?

Reply:
After rereading this, I came up with two suggestions:

1st => Leave it as it is, because in 5.1 of [Setting up a Microsoft Entra ID Enterprise application](https://docs.dataminer.services/user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Configuring_SAML/SAML_using_Entra_ID.html?tabs=tabid-1%2Ctabid-3#setting-up-a-microsoft-entra-id-enterprise-application) we say what the Entity ID must be and give an example.

2nd => Replace [ENTITYID] with the IP address or the DNS name of your DataMiner System. The specified URL(s) must match the Entity ID you specified while setting up the Microsoft Entra ID Enterprise application.

Reply:
Can the entity ID be more than one URL then, since you write "The specified URL(s)"?

Reply:
No, it just needs to be one single string

Reply:
In that case, how about this? 
```suggestion
1. Replace [ENTITYID] with the IP address or the DNS name of your DataMiner System. This URL must be the same as the *Entity ID* you specified while setting up the Microsoft Entra ID Enterprise application.
```